frontend/src/App.jsx

```diff
@@ -89,6 +75,7 @@ function App() {
 
   return (
     <div className="App">
+      <Notifier />
```

Comment:
@NowGoody can you please remove other Notifier appearences accross the app, as we now have this root one, because now it seems to create double notifications and even some unexpected crashes.
(LendingForm.js, TelegramNotificationModal.jsx, DontMiss.jsx, Home.jsx)

Reply:
Alright I will do that now